### PR TITLE
Improving codacy score

### PR DESCRIPTION
**Pull request by AndyScherzinger**
as discussed on IRC fixing codacy warnings

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
PR fixes 26 warnings

File: app/src/test/java/com/nextcloud/talk/ExampleUnitTest.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
  */
 public class ExampleUnitTest {
     @Test
-    public void addition_isCorrect() throws Exception {
+    public void additionIsCorrect() throws Exception {
         assertEquals(4, 2 + 2);
     }
 }
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply app/src/test/java/com/nextcloud/talk/ExampleUnitTest.java]: DONE https://github.com/nextcloud/talk-android/pull/103/commits/9a07f28815735fc3c1dad6b7ec66cd0ef302d4bd :smiley:

**Comment by mario** [comment]
Thanks.

**Comment by AndyScherzinger** [comment]
You are very welcome ❤️

**Pull request merged**